repos/coreclr.proj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildArguments>$(Platform) $(Configuration) skiptests -PortableBuild=false </BuildArguments>
+    <BuildArguments>$(Platform) $(Configuration) -PortableBuild=$(PortableBuild) skiptests</BuildArguments>
```

Comment:
Looks like order is important here as CI is failing. Switch this back to skiptests before.